src/Datasource/EntityTrait.php

```diff
@@ -1231,7 +1231,7 @@ public function source($alias = null)
      */
     public function __toString()
     {
-        return json_encode($this, JSON_PRETTY_PRINT);
+        return json_encode($this, JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE);
```

Comment:
This could change/break existing applications that are relying on string casts to be usable in javascript code. I'm not sure that unescaped unicode will work in all javascript engines.